Userland/Libraries/LibGUI/TextDocument.h

```diff
@@ -259,4 +259,16 @@ class ReplaceAllTextCommand final : public GUI::TextDocumentUndoCommand {
     String m_action_text;
 };
 
+class IndentSelection : public TextDocumentUndoCommand {
+public:
+    IndentSelection(TextDocument&, size_t const&, TextRange const&);
```

Comment:
Please only omit parameter names in declarations if they are clear from the type name, `size_t` could be anything